canonical-web-and-design/ubuntu.com
# Fix alignment and spacing issues

## Done

- updated `p-heading-icon` markup and styling to correct alignment of images
- updated markup on signup forms to reduce padding between checkboxes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit the following pages, and see that the padding between checkboxes matches the design [here](https://github.com/canonical-web-and-design/vanilla-framework/issues/2392#issuecomment-518601399)
  - [x] /blog
  - [x] /download/desktop/thank-you

- Visit the following pages and ensure that elements with header icons don't look misaligned:
  - [x] /blog/topics/design
  - [x] /blog/topics/juju
  - [x] /blog/topics/maas
  - [x] /blog/topics/robotics
  - [x] /blog/topics/snappy
  - [x] /download/cloud
  - [x] /download/desktop
  - [x] /download/iot
  - [x] /financial-services
  - [x] /internet-of-things/appstore
  - [x] /internet-of-things/digital-signage
  - [x] /internet-of-things
  - [x] /internet-of-things/robotics
  - [x] /kubeflow/features
  - [x] /kubernetes
  - [x] /kubernetes/install
  - [x] /kubernetes/managed
  - [x] /openstack/consulting
  - [x] /openstack/features
  - [x] /openstack/managed
  - [x] /openstack/what-is-openstack
  - [x] /pricing
  - [x] /public-cloud


## Issue / Card

Fixes #5650 & #5652
